src/main/cpp/src/HostTableJni.cpp

```diff
@@ -26,8 +26,8 @@
 #include <cuda_runtime_api.h>
 
 #include <algorithm>
-#include <iterator>
 #include <memory>
+#include <numeric>
```

Comment:
Change to fix compile error. The error is due to not including the necessary header here but such header was included indirectly somewhere else.